### PR TITLE
Colinmacdonald/test build readme

### PR DESCRIPTION
**Pull request by colinmacdonald**
- [x] Adds project files
  - .gitignore
  - Makefile
  - LICENSE
  - component.json
  - package.json
- [x] Adds README.md
- [x] Adds tests and travis config
  - test/index.html
  - test/index.js
  - .travis.yml
- [x] Removes bower from grunt build task

### CODE REVIEW

File: Gruntfile.js [hunk 1]
```diff
@@ -0,0 +1,152 @@
...
+
+    // To prevent having to write a custom build.js file, we take advantage
+    // of their lack of escaping in order to build up the widgets namespace.
+
+    // Yeah, this is hacky.
+    // In the following code string's context, `this` is the window object.
+    var exportName = '"];' +
+      'this.goinstant = this.goinstant || {};' +
+      'this.goinstant.widgets = this.goinstant.widgets || {};' +
+      'this.goinstant.widgets["Colors Common';
```
Comment (nealstewart) [inline Gruntfile.js]: This doesn't need to be able to build for prod

Comment (nealstewart) [inline Gruntfile.js]: You can just remove the entire prod task

Comment (colinmacdonald) [inline Gruntfile.js]: oh i just c and p'd from usercache, it will need the same thing removed then.

**Comment by nealstewart** [comment]
LGTM

**Pull request merged**